### ISSUE TITLE
Roadmap

### ISSUE THREAD

**Issue by andrewisen** [andrewisen/bim-whale#1]
__Things todo:__

- Improve parser (more specific `parse-step-instance-attributes.ts`)
- Add support for [IfcProductType](https://standards.buildingsmart.org/IFC/RELEASE/IFC2x3/FINAL/HTML/ifckernel/lexical/ifctypeproduct.htm)
- ~~Add dist (JS Bundle File)~~
- ~~Add web example (using the dist)~~
- Implement Web Workers
- ~~Add more samples~~
- Update Docs/Wiki
- Fix spelling/language
- etc. etc

__Will there be added features?__
> The parser is very simple... it doesn't do much. Will there be added features?

In short, no. The goal of this repo is to provide learning resources.
I do not intend to build "the best parser in the world".... not yet :)
